port/unix/j9nlshelpers.c

```diff
@@ -117,7 +117,7 @@ nls_determine_locale(struct OMRPortLibrary *portLibrary)
 	}
 #endif /* defined (J9ZOS390) */
 #endif  /* defined(LINUX) || defined(OSX) */
-	if (lang != NULL && strcmp(lang, "POSIX") && strcmp(lang, "C"))
+	if (lang != NULL && strcmp(lang, "POSIX") && strcmp(lang, "C") && strcmp(lang, "UTF-8"))
```

Comment:
Can you add the `{` & `}` to this if statement?  It will make the code clearer

Reply:
Added the braces and added some more parenthesis to the conditionals.